2-mandatory/6-budgets.js

```diff
@@ -11,6 +11,12 @@ Should give return the answer of 62600.
 **/
 
 function getBudgets(peopleArray) {
+    let sum = 0;
```

Comment:
Very good. If you are feeling brave, you can also use `reduce()`
Something like this:

```
return peopleArray.reduce((total, person) => total + person.budget, 0);
```